core/scripts/run-tests.sh

```diff
@@ -3,6 +3,59 @@
 /**
  * @file
  * This script runs Backdrop tests from command line.
+ *
+ *
+ *  Run Backdrop tests from the shell.
+ *
+ *  Usage:        {$args['script']} [OPTIONS] <testClassNames>
+ *  Example:      {$args['script']} --url="http://backdrop" BootstrapPageCacheTestCase
+ *
+ *  All arguments are long options.
+ *
+ *  --help      Print this page.
```

Comment:
Let's add a line for:
```
@code
```
Just above the --help line so these options will be formatted without losing their white-space.